FRW-1443 Added note about data regenerate for search and storage (#1685)
spryker/spryker-docs: _includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

The key name follows this format: `kv:{store}:{locale}:{resource-type}:{key}`.

Note that the key name can be different, depending on the configuration of the project.
Below is a list of keys, taking into account the default configuration out of the box.

Example list of keys where `xxx` - is a store name: 

- `kv:availability:xxx:*`
- `kv:price_product_abstract:xxx:*`
- `kv:price_product_abstract_merchant_relationship:xxx:*`
- `kv:price_product_concrete:xxx:*`
- `kv:price_product_concrete_merchant_relationship:xxx:*`
- `kv:product_abstract:xxx:*`
- `kv:product_abstract_category:xxx:*`
- `kv:product_abstract_option:xxx:*`
- `kv:product_abstract_relation:xxx:*`
- `kv:product_concrete_measurement_unit:xxx:*`
- `kv:product_concrete_product_offer_price:xxx:*`
- `kv:product_concrete_product_offers:xxx:*`
- `kv:product_label_dictionary:xxx:*`
- `kv:product_offer:xxx:*`
- `kv:product_offer_availability:xxx:*`
- `kv:category_node:xxx:*`
- `kv:category_tree:at:*`
- `kv:cms_block:xxx:*`
- `kv:cms_page:xxx:*`
- `kv:merchant:xxx:*`
- `kv:price_product_abstract_merchant_relationship:xxx:*`
- `kv:store:xxx`

Also appear new keys for dynamic store feature:

> `kv:store_list:` and delete store name XXX values in stores. 

Author: Sorry. Only for DS ON, I got confused and wrote it wrong.

Please, check `deploy.dev.dynamic-store.yml` file for more details.

## Data migration 

> You don't need to migrate data in the database for the dynamic store feature. Only changes in the search and storage engine are necessary.

Reviewer: is it true that we don't need to "migrate data"? Not a comment, just a question

Author: We don't have big changes at the base level. All relations used store_id. these relations were present before. Added a few tables for save currency, locale, country in a separate table. And it's all

> Propel migration will be executed automatically and added new tables for dynamic store feature and populate new tables with data via data import.

Reviewer 2: ```suggestion
Propel migration executes automatically, adds new tables for the dynamic store feature, and populates new tables with data via data import.
```
Propel the tool right? Not a misspelling of proper?

> For correct work of dynamic store feature you need to add new data structure for search and storage engines.

Reviewer 2: ```suggestion
To make the dynamic store feature work correctly, you need to add new data structures for search and storage engines.
```

> First of all, you need to truncate all data from search and storage engines. 

Reviewer 2: ```suggestion
You first need to truncate all data from search and storage engines. 
```

Please see below for more details in the section **"Truncate data from search and storage engines"**.

> Next step is run sync:data command for populate new data structure for search and storage engines.

Reviewer 2: ```suggestion
Next, run the `sync:data` command to populate the new data structure for search and storage engines.
```

```bash
vendor/bin/console sync:data
```

All data will be populated in search and storage engines.

> {% info_block warningBox "Verification" %}

Reviewer 2: ```suggestion
{% info_block warningBox "Verification" %}
```

Should this be removed?

### Truncate data from search and storage engines

#### Storage engine

> Since Dynamic Store feature uses Redis as a key-value store for each region instead of store. If using same Redis instance for region, you need to clear all data from Redis for existing stores.

Reviewer 2: ```suggestion
The Dynamic Store feature uses Redis as a key-value store for each region instead of store. If you are using the same Redis instance for a region, you need to clear all data from Redis for existing stores.
```

The data is stored in the following format: `kv:{resource-type}:{store}:{locale}:{key}`.
 

>  For remove data from Redis you can use `redis-cli` command:

Reviewer 2: ```suggestion
 To remove data from Redis, use the `redis-cli` command:
```

 
 ```bash
 redis-cli KEYS "*:xxx:*" | xargs redis-cli DEL
```

{% info_block warningBox  %}
Note that the key name can be different, depending on the configuration of the project.
Below is a list of keys, taking into account the default configuration out of the box.
{% endinfo_block %}

{% info_block warningBox "Verification" %}

> Example list of keys where `xxx` - is a store name: 

Reviewer: please add an example how store key should look like

> Example list of keys where `xxx` - is a store name: 

Reviewer 2: ```suggestion
Example list of keys where `xxx` is a store name: 
```

- `kv:availability:xxx:*`
- `kv:price_product_abstract:xxx:*`
- `kv:price_product_abstract_merchant_relationship:xxx:*`
- `kv:price_product_concrete:xxx:*`
- `kv:price_product_concrete_merchant_relationship:xxx:*`
- `kv:product_abstract:xxx:*`
- `kv:product_abstract_category:xxx:*`
- `kv:product_abstract_option:xxx:*`
- `kv:product_abstract_relation:xxx:*`
- `kv:product_concrete_measurement_unit:xxx:*`
- `kv:product_concrete_product_offer_price:xxx:*`
- `kv:product_concrete_product_offers:xxx:*`
- `kv:product_label_dictionary:xxx:*`
- `kv:product_offer:xxx:*`
- `kv:product_offer_availability:xxx:*`
- `kv:category_node:xxx:*`
- `kv:category_tree:xxx:*`
- `kv:cms_block:xxx:*`
- `kv:cms_page:xxx:*`
- `kv:merchant:xxx:*`
- `kv:price_product_abstract_merchant_relationship:xxx:*`
- `kv:store:xxx`

{% endinfo_block %}

> But recomedation flash all data from Redis for instance and populate it again.

Reviewer: ```suggestion
But recommendation is to flash all data from the Redis by instance and populate them again.
```

> But recomedation flash all data from Redis for instance and populate it again.

Reviewer 2: ```suggestion
It's recommended to flash all data from Redis by instance and populate them again.
```

Author: @AlexSlawinski Please review again this part.

### Search engine

> For dynamic store feature out of the box changed indexes prefix for all stores. So data from old indexes not available for new stores.

Reviewer: ```suggestion
For the dynamic store feature out of the box index prefixes were changed for all stores. So data from old indexes are not available for new stores.
```

Reviewer 2: > 

```suggestion
The out of the box dynamic store feature changed the prefixes of indexes for all stores. Due to this, data from old indexes is no longer available for any new stores.
```

> If you are using Elasticsearch, following indexes are available in the standard configuration (example: `xxx` -  store name): 

Reviewer 2: ```suggestion
If you are using Elasticsearch, the following indexes are available in the standard configuration (example: `xxx` -  store name): 
```

- `spryker_xxx_merchant`.
- `spryker_xxx_page`.
- `spryker_xxx_product-review`.
- `spryker_xxx_return_reason`.

> For delete indexes you can use `curl` command. For example for delete `spryker_xxx_page` index: 

Reviewer 2: ```suggestion
To delete indexes, you can use the `curl` command. For example, to delete the `spryker_xxx_page` index: 
```
